### PR TITLE
Master add sol imgui bindings

### PR DESCRIPTION
**Pull request by projecteon**
Added bindings for table column flag checks.
The sample code for context menu in table columns use these.

### CODE REVIEW

File: src/plugins/lua/contrib/imgui/sol_ImGui.h [hunk 1]
```diff
@@ -1494,9 +1494,12 @@ namespace sol_ImGui
...
 
-	inline int TableGetColumnCount()                                                                    { return ImGui::TableGetColumnCount(); }
+	inline int TableGetColumnCount()																	{ return ImGui::TableGetColumnCount(); }
+	inline int TableGetColumnFlags(int column_n)                                                        { return ImGui::TableGetColumnFlags(column_n); }
 	inline std::string TableGetColumnName()                                                             { const char* name = ImGui::TableGetColumnName(); if (name) return std::string(name); return std::string(); }
 	inline std::string TableGetColumnName(int column_n)                                                 { const char* name = ImGui::TableGetColumnName(column_n); if (name) return std::string(name); return std::string(); }
+	inline bool TableGetColumnHasFlag(ImGuiTableColumnFlags flag)										{ return (ImGui::TableGetColumnFlags() & flag) != 0; }
+	inline bool TableGetColumnHasFlag(int column_n, ImGuiTableColumnFlags flag)							{ return (ImGui::TableGetColumnFlags(column_n) & flag) != 0; }
```
Comment (brainiac) [inline src/plugins/lua/contrib/imgui/sol_ImGui.h:1502]: please use spaces instead of tabs for aligning the function bodies

**Review by brainiac — CHANGES_REQUESTED** [review]
please use spaces instead of tabs for aligning the function bodies

**Comment by projecteon** [comment]
Updated, 
Sorry, Im used to having tools (ie editorconfig) that does this automatically for me.

**Comment by brainiac** [comment]
turn on whitespace markers so you can see :)

We do have editor config, the indentation should be tabs, but the alignment for the function bodies is spaces. indentation and alignment are two different things here.

I'll fix.t

**Pull request merged**